bin/ConcordanceCheck.sh

```diff
@@ -184,7 +184,7 @@ do
 		arrayId=$(sed 1d "${sampleSheet}" | awk 'BEGIN {FS="\t"}{print $1}')
 		arrayVcf="${arrayId}.FINAL.vcf"
 		ngsId=$(sed 1d "${sampleSheet}" | awk 'BEGIN {FS="\t"}{print $2}')
-		ngsVcf="${ngsId}.final.vcf.gz"
+		ngsVcf="${ngsId}"
```

Comment:
heb je dit getest? de ngsId is de sample naam zonder de extentie .final.vcf.gz, je kan toch alleen een zcat doen op een file? zie regel 200